apps/condo/domains/property/components/panels/Builder/MapConstructor.ts

```
@@ -357,97 +347,75 @@
+        }
     }
 
-    get parking (): BuildingSection[] {
-        return this.map.parking
+    get availableUnitTypes (): BuildingUnitSubType[] {
+        if (this.viewMode === MapViewMode.section) {
+            return SECTION_UNIT_TYPES as BuildingUnitSubType[]
+        } else if (this.viewMode === MapViewMode.parking) {
+            return PARKING_UNIT_TYPES as BuildingUnitSubType[]
+        }
     }
```

Biome (1.9.4)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)